documentation/CODESPEED.rst

```diff
@@ -16,7 +16,7 @@ Using Codespeed 0.8.1
 
 To download it, go to https://github.com/tobami/codespeed and use the
 downloads button on the right side to get the latest stable
```

Comment:
Likewise, this file can go as well since we've never actually used codespeed.

Reply:
Dropped.